test/crypto/elliptic.js

```diff
@@ -151,63 +151,49 @@ describe('Elliptic Curve Cryptography', function () {
       }
       done();
     });
-    it('Creating KeyPair', function () {
-      const names = ['p256', 'p384', 'p521', 'secp256k1', 'curve25519', 'brainpoolP256r1', 'brainpoolP384r1', 'brainpoolP512r1'];
+    it('Creating KeyPair light build', function () {
+      const names = ['p256', 'p384', 'p521', 'curve25519'];
```

Comment:
Could you either skip this test when the next one is not skipped, or (probably cleaner) merge them into one test but just select a different `names` array depending on the build?

Reply:
We could separate test cases for light-build and full-build to different files in the future. Then different tests would be easier to understand.

Reply:
I'm not entirely sure that would help; by having an explicit `if` statement inside the test, it becomes very obvious what the difference between the lightweight and the full build is expected to be. Whereas if you have a similar test in two different files, you need to manually compare them to tell the difference.